### PR TITLE
ref(crons): Support writing is_muted

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Support writing is muted. Follow up after https://github.com/getsentry/sentry/pull/61467

### CODE REVIEW

File: src/sentry/monitors/validators.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ class MonitorValidator(CamelSnakeSerializer):
         default="active",
         help_text="Status of the monitor. Muted monitors do not generate events or notifications.",
     )
+    is_muted = serializers.BooleanField(help_text="Disable creation of monitor incidents")
```
Comment (50-Course) [inline src/sentry/monitors/validators.py:252]: 👍🏼

**Review by 50-Course — APPROVED** [review]
Thank you for this MR. Functional implementation looks good to me. A few tests, however is failing due to the change.

Ref: `tests/sentry/monitors/endpoints/test_organization_monitor_index.py::CreateOrganizationMonitorTest`

**Review by cmanallen — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
I think it will be okay. I am planning to bring back another test where we'll allow backwards compatability at the validator and serializer level, at which point I'll add a new test.

File: tests/sentry/monitors/endpoints/test_organization_monitor_details.py [hunk 2]
```diff
@@ -133,26 +133,24 @@ def test_slug_same(self):
     def test_can_mute(self):
         monitor = self._create_monitor()
         resp = self.get_success_response(
-            self.organization.slug, monitor.slug, method="PUT", **{"status": "muted"}
+            self.organization.slug, monitor.slug, method="PUT", **{"is_muted": True}
```
Comment (evanpurkhiser) [inline tests/sentry/monitors/endpoints/test_organization_monitor_details.py:136]: Thi should be `isMuted` I think?

**Pull request merged**